llpc/lower/llpcSpirvLower.cpp

```diff
@@ -188,6 +189,9 @@ void SpirvLower::addPasses(Context *context, ShaderStage stage, lgc::PassManager
     passMgr.addPass(SpirvLowerRayTracingIntrinsics());
 #endif
 
+  // Lower SPIR-V CFG merges
+  passMgr.addPass(SpirvLowerCfgMerges());
```

Comment:
I think we need to run this pass after function inlining for now. In case there is a function call in break block, and there are convergent operations in the callee function, we also need to ensure the break block not pulled out of the loop.

Reply:
I agree with this, it is just proving quite difficult as the inliner passes cannot handle functions that take labels are arguments.  I tried to use BlockAddress in this implementation, but the inliner then does not see these are viable unless they are part of a CallBr.  So I need to think a bit more about strategy for carrying data through from SPIRV.

Reply:
I found a suitable scheme to rework this.